pkgs/development/tools/phantomjs2/default.nix

```diff
@@ -104,6 +104,10 @@ in stdenv.mkDerivation rec {
     rm -r ../__nix_qt5__
   '';
 
+  postFixup = ''
```

Comment:
@ttuegel what value will QT_QPA_PLATFORM_PLUGIN_PATH have when installed. Is it possible to set a reasonable value in `nix-shell` (by using `shellHook`)?